python/pysmurf/client/util/smurf_util.py

```diff
@@ -825,15 +841,14 @@ def read_stream_data(self, datafile, channel=None,
         # data structures will be build based on that
         first_read = True
 
-        #with open(datafile, mode='rb') as file:
+        # Maximum number of elements to read before appending
+        n_max = 1024
```

Comment:
Make a function argument instead of a hard-code if it may ever need to be adjusted.

Reply:
This is done in a later commit.